package.json

```diff
@@ -35,6 +35,7 @@
     "watch": "1.0.2"
   },
   "dependencies": {
-    "lodash.curry": "4.1.1"
+    "lodash.curry": "4.1.1",
+    "riteway": "^2.0.3"
```

Comment:
This might be part of `devDepencies` instead, and maybe `tape` could be removed from `devDepencies` as it's already a dependency of `riteway`. 😅

It will keep the package leaner when in prod 😎

Reply:
Yes.